nautobot/extras/models/groups.py

```diff
@@ -50,6 +50,7 @@ class StaticGroup(PrimaryModel):
 
     clone_fields = ["content_type"]
     is_static_group_associable_model = False
+    is_saved_view_model = False
```

Comment:
Why not support saved views for Static Groups?

Reply:
This branch was outdated and didnt have `StaticGroupUIViewSet` yet. Had to do it to make the factory not throw an error.